samples/contrib/aws-samples/sagemaker_debugger_demo/debugger-training-pipeline.py

```diff
@@ -9,91 +9,130 @@
 
 
 cur_file_dir = os.path.dirname(__file__)
-components_dir = os.path.join(cur_file_dir, '../../../../components/aws/sagemaker/')
+components_dir = os.path.join(cur_file_dir, "../../../../components/aws/sagemaker/")
+
+sagemaker_train_op = components.load_component_from_file(
+    components_dir + "/train/component.yaml"
+)
 
-sagemaker_train_op = components.load_component_from_file(components_dir + '/train/component.yaml')
 
 def training_input(input_name, s3_uri, content_type):
     return {
         "ChannelName": input_name,
         "DataSource": {"S3DataSource": {"S3Uri": s3_uri, "S3DataType": "S3Prefix"}},
-        "ContentType": content_type
+        "ContentType": content_type,
     }
 
 
 def training_debug_hook(s3_uri, collection_dict):
     return {
-        'S3OutputPath': s3_uri,
-        'CollectionConfigurations': format_collection_config(collection_dict)
+        "S3OutputPath": s3_uri,
+        "CollectionConfigurations": format_collection_config(collection_dict),
     }
 
 
 def format_collection_config(collection_dict):
     output = []
     for key, val in collection_dict.items():
-        output.append({'CollectionName': key, 'CollectionParameters': val})
+        output.append({"CollectionName": key, "CollectionParameters": val})
     return output
 
 
-def training_debug_rules(rule_name, parameters):
+def training_debug_rules(rule_name, parameters, region):
+    debugger_rule_accounts = {
```

Comment:
you can use sagemaker SDK to get the image URI 
`image_uris.retrieve(framework='debugger', region=args.sagemaker_region, version='latest')`